Documentation/Maintainers/FluidViewHelper.rst

```diff
@@ -0,0 +1,87 @@
+.. include:: /Includes.rst.txt
+..  _fluid-viewhelper-reference-generation:
+
+=====================================
+Fluid ViewHelper reference generation
+=====================================
+
+The :ref:`Fluid ViewHelper Reference <t3viewhelper:start>` gets automatically
+updated by changes to the according ViewHelper classes in the
+`TYPO3 Core <https://github.com/TYPO3/typo3>`__ and
+the package `Fluid Rendering Engine <https://github.com/TYPO3/Fluid>`__.
+
+Context changes need to be made in those two repositories. For contributions
+to the TYPO3 Core, follow the
+:ref:`TYPO3 Contribution Guide - Core Development <t3contribute:recommended-reading>`.
+
+
+..  _fluid-viewhelper-reference-generation-schema:
+
+The Fluid ViewHelper schema generator
+=====================================
+
+The PHP files defining the ViewHelpers are transferred into :file:`schema.xsd`
+files. The package `fluid-schema-generator <https://github.com/TYPO3/Fluid.SchemaGenerator>`__
+is responsible for this step. So if information is already missing in the
+:file:`schema.xsd`, adjustments need to be made here. The information from the
+:file:`schema.xsd` files is also used by IDEs for context sensitive help in
+editors etc.
+
+You can regenerate the :file:`schema.xsd` files for local testing like this:
+
+..  code-block:: bash
+
+    git clone git@github.com:typo3/typo3.git core
+    cd core
+    composer require -o -n --no-progress typo3/fluid-schema-generator "^2.1"
```

Comment:
Should we use a version here, that we may forget to update? Maybe just use "*" to use the most recent one...?